src/data/javascript-quiz.ts

```diff
@@ -1968,6 +1968,16 @@ const javascriptQuiz = [
     Explanation:
       "An IIFE or Immediately Invoked Function Expression, is used to declare a private variable and to avoid polluting the global environment. If a function is declared with an IIFE by using a parenthesis around it, the variable declared inside it becomes private and its value can be accessed using a getter function inside the original function.",
     Link: "https://developer.mozilla.org/en-US/docs/Glossary/IIFE"
+  },
+  {
+    Question: "In JavaScript, which is an example of macrotask?",
+    Answer: "macrotask examples: setTimeout(), setInterval()",
+    Distractor1: "promise",
+    Distractor2: "setTimeout()",
```

Comment:
remember that distractors are for wrong answers only

```suggestion
    Distractor2: "split()",
```